setup.py

```diff
@@ -40,11 +40,19 @@
                        "{}/lib/".format(ucc_home)]
 plugin_libraries    = ["ucp", "uct", "ucm", "ucs", "ucc"]
 
+if '--oss' in sys.argv:
+  sys.argv.remove('--oss')
+  plugin_sources += ["src/torch_ucc_init_oss.cpp"]
+  name = "torch_ucc_oss"
```

Comment:
We do not need to change the name to torch_ucc_oss - the name can and should remain same. regardless of which cpp file is used.

Reply:
fixed